include/gauge_field_order.h

```diff
@@ -1552,14 +1549,10 @@ namespace quda {
           for (int j = 0; j < N; j++) copy(tmp[i * N + j], reinterpret_cast<Float *>(&vecTmp)[j]);
         }
 
-        real phase = 0.;
```

Comment:
just wondering as I had looked at that warning before: For `!hasPhase` we had `phase=0.` in the code and now it seems that has gone. I have not looked at it in detail but did not want to forget about it.

Reply:
Yeah, it doesn’t affect anything.  For the non-phased reconstruct types, it doesn’t use the phase value at all so it doesn’t matter what this is set to.  I checked that the ctest staggered and Wilson tests all still pass with a full reconstruct build.